src/Employee.java

```diff
@@ -0,0 +1,60 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Employee {
+
+    private int age;
+    private String name;
+    private String address;
+
+    /*
+        Throwing unchecked exception. Object will not be created.
+     */
+    public Employee(int age, String name, String address) {
+        if (age < 18) {
+            throw new IllegalArgumentException("Age is not valid");
+        }
+        this.address = address;
```

Comment:
add catch block for IllegalArgumentException- and see if object will be created - that was the discussion as I remember

Reply:
I had another constructor for that case, but added catch block here too.